### ISSUE TITLE
/shipinfo command calculates DPS (damage per second) incorrectly

### ISSUE THREAD

**Issue by cswimr** [GalaxyRoblox/Issues-Public#90]
The `/shipinfo` command currently miscalculates DPS, because it disregards the following information:
- Damage type (Kinetic / Laser / Plasma)
- Custom reloads
- Custom damage values
- Turret overrides (if those are still a thing? the thing I'm referencing for this is from 2023, might be changed now)

**Comment by Wingysam** [comment]
I believe custom reloads/damages for spinals are considered as of the spinal rewrite? Damage type is complicated because it varies at runtime. Like what if we made a module that gives everyone on your team plasma damage?

Turret overrides are no longer a thing, every turret has its stats in Turrets.yml.

**Comment by cswimr** [comment]
> Damage type is complicated because it varies at runtime. Like what if we made a module that gives everyone on your team plasma damage?

I don't think this matters. `/shipinfo` should only list what the ship can do on its own, and without modules, as they won't always be active. It should be as accurate as possible to the ship, ignoring its modules. For example, Deity's top speed is listed as 60, but it has a top speed of 150 with its module enabled.

**Comment by Wingysam** [comment]
Will be fixed by #92
